src/omni_epd/displays/inky_display.py

```diff
@@ -48,111 +52,71 @@ def __init__(self, deviceName, config):
         elif(dType == 'what'):
             deviceObj = self.load_display_driver(self.pkg_name, 'what')
             self._device = deviceObj.InkyWHAT(dColor)
+        elif(dType == 'impression'):
+            deviceObj = self.load_display_driver(self.pkg_name, 'inky_uc8159')
+            self._device = deviceObj.Inky(dColor)
+            self.mode = dColor = 'color' # default to color for impression

```

Comment:
```suggestion
            self._device = deviceObj.Inky()
            dColor = 'color' # default to color for impression
```
While it would be nice if all displays defaulted to their native color depth, setting mode here keeps other modes from being set in the INI.